src/pages/PreviousRuns.js

```diff
@@ -1,8 +1,244 @@
+      submissionTimestamp: '2019-07-12T22:24:15.591Z'
+    }
+  ]
+
+  const sortedPreviousRuns = _.orderBy(sort.field, sort.direction, previousRuns)
+
+  const firstPageIndex = (pageNumber - 1) * itemsPerPage
+  const lastPageIndex = firstPageIndex + itemsPerPage
+  const paginatedPreviousRuns = sortedPreviousRuns.slice(firstPageIndex, lastPageIndex)
+
+
+  return h(Fragment, [
+    headerBar(),
+    div({ style: { margin: '4em' } }, [
+      h2(['Previous Runs']),
+      div(['Remember to turn off your Cromwell App in Terra once you are done to prevent incurring costs.']),
+      div({ style: { marginTop: '1em', height: tableHeight({ actualRows: paginatedPreviousRuns.length, maxRows: 12.5 }), minHeight: '10em' } }, [
+        h(AutoSizer, [
+          ({ width, height }) => h(FlexTable, {
+            'aria-label': 'previous runs',
+            width, height, sort,
+            rowCount: paginatedPreviousRuns.length,
+            noContentMessage: 'Nothing here yet! Your previously run workflows will be displayed here.',
+            hoverHighlight: true,
+            columns: [
+              {
+                size: { basis: 500 },
+                field: 'workflowName',
+                headerRenderer: () => h(Sortable, { sort, field: 'workflowName', onSort: setSort }, ['Workflow Name']),
+                cellRenderer: ({ rowIndex }) => {
+                  return h(TextCell, [paginatedPreviousRuns[rowIndex].workflowName])
+                }
+              },
+              {
+                size: { basis: 150, grow: 0 },
+                field: 'workflowStatus',
+                headerRenderer: () => h(Sortable, { sort, field: 'workflowStatus', onSort: setSort }, ['Status']),
+                cellRenderer: ({ rowIndex }) => {
+                  return h(TextCell, [paginatedPreviousRuns[rowIndex].workflowStatus])
+                }
+              },
+              {
+                size: { basis: 80, grow: 0 },
+                field: 'workflowInputs',
+                headerRenderer: () => 'Inputs',
+                cellRenderer: ({ rowIndex }) => {
+                  return div({ style: { width: '100%', textAlign: 'center' } }, [
+                    h(Link, { onClick: () => setViewInputsId(rowIndex) }, ['View'])
+                  ])
+                }
+              },
+              {
+                size: { basis: 250, grow: 0 },
+                field: 'submissionTimestamp',
+                headerRenderer: () => h(Sortable, { sort, field: 'submissionTimestamp', onSort: setSort }, ['Submitted']),
+                cellRenderer: ({ rowIndex }) => {
+                  return h(TextCell, [Utils.makeCompleteDate(paginatedPreviousRuns[rowIndex].submissionTimestamp)])
+                }
+              }
+            ]
+          })
+        ])
+      ]),
+      !_.isEmpty(sortedPreviousRuns) && div({ style: { bottom: 0, position: 'absolute', marginBottom: '1.5rem', right: '4rem' } }, [
```

Comment:
TOL: I wonder the `isEmpty` could be replaced by something like "if length is less than 10", since there's no point showing pagination options if there are not enough rows for a second page? Maybe that's an enhancement too far and would actually be confusing?

Reply:
I think it might be cleaner to show the pagination even if items are less than 10. It seems more consistent instead of having pagination appear and disappear based on length of previous runs.